docs/Converters/OASIS3TOBIDS.md

```diff
@@ -0,0 +1,166 @@
+
+## Dependencies
+
+If you installed the core of Clinica, this converter needs no further dependencies.
+
+## Downloading OASIS-3
+
+The OASIS-3 to BIDS converter requires the user to have downloaded the OASIS-3 (also called *Longitudinal Neuroimaging, Clinical, and Cognitive Dataset for Normal Aging and Alzheimer’s Disease*) imaging and clinical data. To do so, visit the [OASIS website](http://www.oasis-brains.org/), click on `DATASETS` then `OASIS-3`. For the first access, you have to scroll down to click on `Apply To Access OASIS Data` in the bottom of the next "Data Use Agreement" section and follow the procedure. Then, you can download the data via XNAT (if you click on `Browse Data`).
+
+The data needs to be formatted before downloading by following the step-by-step procedure below.
+
+1. From the menu on the right of the "Subjects" data table, click on the "Edit Columns" option.
+
+![](../img/OASIS3_download/edit_columns.png)
+
+2. Select "Education (Subject)" from the "Available Fields".
+
+![](../img/OASIS3_download/select_education.png)
+
+3. Click on the arrow to send it to the left, to "Current Fields".
+
+![](../img/OASIS3_download/add_education.png)
+
+4. From the menu on the right of the data table, load the "MR sessions" table.
+
+![](../img/OASIS3_download/add_mr_sessions.png)
+
+5. From the menu on the right of the loaded "MR Sessions" data table, click on the "Edit Columns" option, select "Age (MR Sessions)" from the "Available Fields" and click on the arrow to send it to the left, to "Current Fields".
+
+![](../img/OASIS3_download/add_age.png)
+
+6. Select "Age (MR Sessions)" from the "Current Fields" and click on the arrow to send it to the right, to "Available Fields".
+
+![](../img/OASIS3_download/remove_age.png)
+
+7. Check that the Age column in the "MR Sessions" table contains decimal values and not integers.
+
+![](../img/OASIS3_download/check_age.png)
+
+8. From this "MR Sessions" table, click on the "Join to" option.
+
+![](../img/OASIS3_download/join_to_mr_sessions.png)
+
+9. Select the "ADRC Clinical Data" table.
+
+![](../img/OASIS3_download/select_clinical_data.png)
+
+10. Do the same "join" by going back to the "Subjects" table, then check the inclusion of the "mmse", "ageAtEntry" and "cdr" columns.
+
+![](../img/OASIS3_download/check_columns.png)
+
+11. In the options, export a "Spreadsheet" of the two tables "Subjects" and "MR Sessions".
+
+![](../img/OASIS3_download/export_spreadsheet.png)
+
+12. Rename the downloaded files to "oasis3_participants.csv" and "oasis3_sessions.csv" respectively.
+
+!!! note
+    You do not have to modify the original folder name before using the converter.
+
+!!! warning
+    We do not currently support the conversion of OASIS-2.
+
+## Modalities supported
+
+[REMOVE?] Currently, the converter converts only the T1-weighted MRI images and clinical data.
```

Comment:
`[REMOVE?]`

what's that?

Reply:
Not being an imaging expert at all, I wanted to make sure I could remove those sentences ;)

Reply:
Well for a non imaging expert, your docs are pretty well written :-)

Do you mind if we perform the last edits ourselves before merging?

Reply:
Aramis team complete the documentation? No worries: feel free to do it. Or don't hesitate to tell me what is missing.